labels.json

```diff
@@ -87,6 +87,14 @@
     "name": "🧹 status: ticket work required",
     "color": "666666"
   },
+  {
+    "name": "🙅 status: discontinued",
```

Comment:
Is this the same as `discarded` https://github.com/EddieHubCommunity/gh-action-open-source-labels/blob/df3b134d2c81e45ba6938abb0e64b1c311907d20/labels.json#L15

Reply:
No No, it's like discontinued is more like it's on halt for now and will be picked by after high priority ones!!

Reply:
The discontinued label is intended for any outstanding issues when a repository is archived or otherwise no longer maintained.